src/idl/tests/annotation.c

```diff
@@ -446,3 +446,50 @@ CU_Test(idl_annotation, parameter_scope)
 #undef M
 #undef A
 #undef TA
+
+
+#define BM(i) "@bit_bound(" i ") bitmask MyBitMask { @position(0) flag0 };"
```

Comment:
This test is faulty I'd say? `@position` is simply ignored, so the value of `flag0` is always `0`.

Reply:
You're right, I've refactored this test a bit and added more test cases, and I fixed the `@position` annotation implementation.